CHANGELOG.md

```diff
@@ -3,7 +3,82 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## 0.43.0
+
+This release contains a number of fixes and enhancements.
+
+### Tooling, SDK, and Runtime
+
+- Add line count to test coverage report (#4843) (authored by @FarisR99)
+- build: run 'make generate' in release container
```

Comment:
We can probably move this to the misc section with a some description about why this was done.

Reply:
We might even drop this, it's not a user-visible change.

Reply:
It is done. :+1: